CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ https://openeo.org/documentation/1.0/developers/api/reference.html#operation/con
 If needed, feature flags are used to allow testing unstable features in development/production,
 without compromising stable operations.
 
+## Unreleased
```

Comment:
Note: we  don't have a release procedure that takes this changelog into account (like we do in python client).
Instead of `## Unreleased ... ### Changed`, just add a header with (approximate merge) date and current version (see `_version.py`). 
Optionally bump the version in that file within the same commit